solana/programs/core-bridge/src/lib.rs

```diff
@@ -20,6 +20,7 @@ pub mod legacy;
 
 mod processor;
 pub(crate) use processor::*;
+pub use processor::WriteEncodedVaaArgs;
```

Comment:
Can we move this to the sdk submodule where the other arg structs are exported?

Reply:
Along with that, can we put the other cpi methods you're using in this export, too? https://github.com/guibescos/wormhole/blob/solana-rewrite/make-some-stuff-public/solana/programs/core-bridge/src/sdk/mod.rs#L40

Reply:
I just meant that the sdk module aggregates all of the cpi calls that an integrator needs, so we can re-export cpi::write_encoded_vaa in here.

I can do that in the next alpha release. Everything looks good thank you